### PR TITLE
Factor installer light package task into a separate arcade task package

### PR DESCRIPTION
**Pull request by mmitche**
This moves CreateLightCommandPackageDrop into a separate arcade task package,
so that the functionality can be reused in some other repos that are not always
building shared framework assemblies.

Ideally this package will eventually factor out all installer task functionality
so it can be shared among all repos. The issue right now is that the way wix installers
are built in the runtime repos, aspnetcore, and installer are all very different.
Installer uses powershell scripts to invoke wix, aspnetcore uses the wix nuget package,
and the runtime repos use the shared framework sdk, which invokes wix using Exec.

I think the right way to do this long term is to use the wix nuget package. This would mean
that this package would reference a version of wix, exposing the tasks, and add its
additional tasks on top of that. Similar functionality could be built for other types of installers.

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/windows/wix.targets [hunk 1]
```diff
@@ -6,7 +6,16 @@
     this file and switch to wixproj. See https://github.com/wixtoolset/issues/issues/5627
   -->
 
-  <UsingTask TaskName="CreateLightCommandPackageDrop" AssemblyFile="$(DotNetBuildTasksSharedFrameworkTaskFile)" />
+  <ItemGroup>
```
Comment (jcagme) [inline src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/windows/wix.targets:9]: Should this (also) exist in src/Microsoft.DotNet.Build.Tasks.Installers/?

Comment (mmitche) [reply src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/windows/wix.targets:9]: Yep I put it in the props file there. The problem is that the evaluation order means you can't reference that props file in the shared framework SDK

Comment (jcagme) [reply src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/targets/windows/wix.targets:9]: Oh yeah, got it

**Comment by mmitche** [comment]
@tmat can you take a look?

**Pull request merged**